source/vocab/relations.ttl

```diff
@@ -51,7 +52,7 @@
     rdfs:label "has part"@en, "har del"@sv;
     rdfs:subPropertyOf :relatedTo;
     owl:inverseOf :isPartOf;
-    owl:equivalentProperty bf2:hasPart, dc:hasPart .
+    owl:equivalentProperty bf2:hasPart, dc:hasPart, relsubtype:hsp .
```

Comment:
We reasonably want to equivalently connect `:isPartOf` to `relsubtype:isp`.

Note that in both cases the (admittedly purely textual) restriction of "when these are the same object category" on the relsubtype properties makes me wonder if the more open and flexible `:hasPart` and `:isPartOf` are better as super-properties of these (and of `relsubtype:inc` and `relsubtype:isi` respectively).

However, in order to enforce such distinctions using OWL, we'd have to make restrictions on these per class. Perhaps keeping this as is and adding something like `:Representation rdfs:subClassOf [ a owl:Restriction ; owl:onProperty :hasPart ; owl:allValuesFrom :Representation ] .` and similarly for `:File` is enough. (And then, presumably, a similar pattern with `owl:allValuesFrom  [ owl:disjointWith :Representation ]` for `:includes` and :`includedIn`.)

Reply:
While we're at it, let's add `sdo:hasPart` and `sdo:isPartOf` equivalents.